internal/util/mqclient/pulsar_consumer.go

```diff
@@ -34,6 +34,7 @@ type PulsarConsumer struct {
 	closeCh    chan struct{}
 	once       sync.Once
 	skip       bool
+	isClosed   bool
```

Comment:
If `Close` could be called by many goroutines at the same time. Please use atomic or sync.once